web/style.css

```diff
@@ -0,0 +1,33 @@
+html, body {
```

Comment:
have you considered sass (or less)?

It is easy to use and I think you would love it

Reply:
I have had enough of "new exciting modern" technologies in the last week, and I see no use for _less_ in such a small scope project.